pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2024 Open Text.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.github.cafapi.ssl</groupId>
+    <artifactId>caf-ssl</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <modules>
+        <module>caf-ssl-dropwizard</module>
+    </modules>
+    <packaging>pom</packaging>
+
+    <parent>
+        <groupId>com.github.cafapi</groupId>
+        <artifactId>caf-common-parent</artifactId>
+        <version>5.0.0-431</version>
+    </parent>
+    
+    <inceptionYear>2024</inceptionYear>
+
+    <properties>
+        <copyrightYear>2024</copyrightYear>
+        <copyrightNotice>Copyright ${copyrightYear} Open Text.</copyrightNotice>
+    </properties>
+
+    <scm>
+        <connection>scm:git:https://github.com/CAFapi/caf-ssl.git</connection>
+        <developerConnection>scm:git:https://github.com/CAFapi/caf-ssl.git</developerConnection>
+        <url>https://github.com/CAFapi/caf-ssl</url>
+    </scm>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>jakarta.xml.bind</groupId>
+                <artifactId>jakarta.xml.bind-api</artifactId>
+                <version>4.0.1</version>
+            </dependency>
```

Comment:
Drop this down below the other two as we want to keep dependency management in alphabetical order.